docs/vgclog.md

```diff
@@ -0,0 +1,268 @@
+    <attribute name="physicalMemory" value="100335456256" />
+    <attribute name="numCPUs" value="28" />
+    <attribute name="architecture" value="amd64" />
+    <attribute name="os" value="Linux" />
+    <attribute name="osVersion" value="3.10.0-1160.el7.x86_64" />
+  </system>
+  <vmargs>
+    <vmarg name="-Xoptionsfile=/java/perffarm/sdks/O11_j9_x64_linux-20201014/sdk/lib/options.default" />
+    ...
+    <vmarg name="-Dcom.ibm.oti.vm.bootstrap.library.path=/java/perffarm/sdks/O11_j9_x64_linux-20201014/sdk/lib/compressedrefs:/java/perffarm/sdk..." />
+    ...
+    <vmarg name="-Xms1024m" />
+    <vmarg name="-Xmx1024m" />
+    ...
+    <vmarg name="-Xverbosegclog:verbosegc.xml" />
+    ...
+  </vmargs>
+</initialized> 
+```
+
+The first set of `<attribute>` elements records the configuration of the garbage collector, such as the GC policy type, configuration of the Java&trade; object heap, and the number of threads that are used for garbage collection. For example, the `GCThreads` attribute records that the garbage collector is configured to use four threads.
+
+The `<system>` section records information about the operating system and available hardware, such as the physical memory, number of CPUs, and operating system type and version. In the example, the VM is running on Linux&reg; amd64 V3.10 and has access to 28 CPUs and over 100 GB.
+
+The `<vmargs>` section records any VM configuration [command-line options](cmdline_specifying.md) (VM arguments) that are specified. The following types of options are recorded:
+
+- non-standard [JVM `-X` options](x_jvm_commands.md) and [JVM `-XX` options](xx_jvm_commands.md). In the example output, the log records the location of the file that contains VM options and definitions as `java/perffarm/sdks/O11_j9_x64_linux-20201014/sdk/lib/options.default`. The verbose GC log option is set to `-Xverbosegclog:verbosegc.xml` to write the verbose GC log output to an XML file. The initial and maximum Java&trade; object heap sizes are both set to 1024 KB by using the `-Xms` and `-Xmx` options.
+- [system property options](d_jvm_commands.md). In the example output, the system property `com.ibm.oti.vm.bootstrap.library.path` is set.
+
+These command-line options can be set by using the command line, or by passing a manifest file, options file, or environment variable to the VM.
+
+After the configurations are recorded in the Initialization section, the verbose GC log begins recording GC activities and details. 
+
+### GC cycles
+
+The start of a GC cycle is recorded by the `<cycle-start>` XML element. The trigger for the start of a GC cycle is captured in a preceding element to the `<cycle-start>` element. A GC cycle or GC increment is triggered for one of the following reasons:
+
+- Because an allocation failure occurs. Allocation failures occur when a request for memory fails the Java&trade; object heap does not have enough memory available. The element `<af-start>` logs an allocation failure trigger. 
```

Comment:
Sorry Mel, I didn't mean "because" was missing at the beginning. Adding it here creates a partial sentence, which isn't good. It is in the following sentence:

Allocation failures occur when a request for memory fails **because** the Java&trade; object heap does not have enough memory available.